src/cast_core/src/HelloWorld.cpp

```diff
@@ -39,6 +39,7 @@ struct HelloWorld::PhaseConfig {
 
 void HelloWorld::run() {
     for (auto&& config : _loop) {
+        BOOST_LOG_TRIVIAL(debug) << "Starting " << this->actorInfo() << " execution";
```

Comment:
bump on the previous comment about mentioning that this doesn't account for rate limiting

Reply:
Done